config/crd/bases/infrastructure.cluster.x-k8s.io_metal3datatemplates.yaml

```diff
@@ -56,6 +56,23 @@ spec:
                 description: MetaData contains the information needed to generate
                   the metadata secret
                 properties:
+                  dnsServersFromIPPool:
+                    description: DNSServersFromPool is the list of metadata items
+                      to be rendered as dns servers.
+                    items:
+                      properties:
+                        key:
+                          description: Key is the metadata key when redendering this
+                            metadata element
+                          type: string
+                        name:
+                          description: Name is the name of the pool to use
```

Comment:
use just 'Name is the name of the pool' or add something after 'to use'

Reply:
this is automatically generated and will be updated with the comment update